_data/rules.yml

```diff
@@ -998,6 +998,66 @@ groups:
           - name: instaclustr/cassandra-exporter
             doc_url: https://github.com/instaclustr/cassandra-exporter
             rules:
+              - name: 'Cassandra Node is unavailable
+                description: 'Cassandra Node is unavailable - {{ $labels.cassandra_cluster }} {{ $labels.exported_endpoint }}'
+                query: 'sum(min_over_time(cassandra_endpoint_active{cassandra_cluster=~".*"}[1m])) by (cassandra_cluster,instance,exported_endpoint) < 1'
+                for: 2m
+                severity: critical
+              - name: 'Cassandra many compaction tasks are pending'
+                description: 'Many Cassandra compaction tasks are pending - {{ $labels.cassandra_cluster }}'
+                query: 'avg_over_time(cassandra_table_estimated_pending_compactions{cassandra_cluster=~".*"}[1m]) > 100'
```

Comment:
No need avg_over_time here. But in that case, `for: 2m` looks fair 👌

Reply:
removed them both